### ISSUE TITLE
Request - new catalog record attribute

### ISSUE THREAD

**Issue by Jegelewicz** [ArctosDB/arctos#8271]
Help us understand your request (check below):
- [ ] search or data request (help with SQL or provide the specific fields you are trying to retrieve for your collection)
- [ ] add a new Github user to the Arctos Users team
- [ ] a new barcode series
- [x] a new term for data entry or management 
- [ ] a change or new report or label
- [ ] update to an Agent (split, merge, or other type of change)
- [ ] bulkload changes (to Agents, Identifiers, etc that are not already available as a bulkloading tool)
- [ ] new feature or otherwise a good idea to make Arctos even better
- [ ] quarantine taxon name and suggested replacement (please supply links)
- [ ] other

**Describe what you're trying to do** 

I have 195 birth dates to add to UWBM:Mamm records. these are zoo or lab animals with known dates of birth and sometimes dates of death that are NOT the same as the date of collection. Because we record "death date" as verbatim preservation date with the date as the value, I think it would be best to record date of birth with the date in the value. In this way, one could determine the numeric age by subtracting one date from the other.

BUT verbatim preservation date is free text - meaning there is all kinds of weirdness in there, so:

I propose two new attributes:

**date of birth** - The date an organism was born. Values can take the form of any date as described in https://handbook.arctosdb.org/documentation/dates.html. http://purl.obolibrary.org/obo/NCIT_C25296

**date of death** - The date an organism ceased to be alive. Values can take the form of any date as described in https://handbook.arctosdb.org/documentation/dates.html. http://purl.obolibrary.org/obo/NCIT_C70810

Values in verbatim preservation date should be converted to date of death and that attribute removed from the code table.

I do not believe the community is ready to record these dates as events, which requires significantly more time and effort. It may be more correct, but is also more resource-intensive. However, I would also be open to two new record event types:

**birth** - Event at which an organism was born. http://purl.obolibrary.org/obo/NCIT_C25155

**death** - Event at which an organism ceased to live. http://purl.obolibrary.org/obo/NCIT_C28554

My concern with this method is that many of the records that exist currently should also include a death event that would be the same as the collection event if we want consistent data.

A timely resolution would be nice as @jebrad would like to get these recorded as part of his RANGES work.

## see also
https://github.com/ArctosDB/arctos/issues/6110
https://github.com/ArctosDB/arctos/issues/1256
https://github.com/ArctosDB/arctos/issues/5600

**Comment by mkoo** [comment]
I agree that it's a record attribute if anything. With that in mind, how would this interact with [numeric age ](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctattribute_type#numeric_age)- add B/D dates as method if known? Or would a single new attribute work? (eg new attribute of "life event"? with Birth/Death Dates) and then using the the fields for Attribute Earliest and Attribute Latest for the dates like Begin and End. Also, how would this be used/ useful in discovery?

**Comment by campmlc** [comment]
We absolutely need this at MSB for our zoo animals. We've been harvesting birth and death dates from ZIMS, and attempting to use them to calculate numeric age with birth date in remarks, but it is an error prone hack that isn't used consistently and takes extra time. It would greatly simplify data entry to capture birth and death dates in an attribute with standard date format.

**Comment by campmlc** [comment]
See for example https://arctos.database.museum/guid/MSB:Mamm:334051

**Comment by mkoo** [comment]
this got buried, transferring to CT committee for polish and implementation
